The svn*.protocol files collide with those provided by kdesdk

--- origsrc/kdesvn-1.3.2/src/kiosvn/CMakeLists.txt	2009-01-07 05:30:07.000000000 -0600
+++ src/kdesvn-1.3.2/src/kiosvn/CMakeLists.txt	2009-07-08 16:54:10.694196500 -0500
@@ -21,7 +21,7 @@ SET_TARGET_PROPERTIES(kio_ksvn
 INSTALL(TARGETS kio_ksvn
     LIBRARY DESTINATION ${PLUGIN_INSTALL_DIR} )
 
-FILE(GLOB PROTFILES *.protocol)
+FILE(GLOB PROTFILES ksvn*.protocol)
 
 INSTALL(FILES ${PROTFILES}
     DESTINATION  ${SERVICES_INSTALL_DIR} )
